modules/client/src/addresslistVoting/internal/client/encoding.ts

```diff
@@ -34,11 +36,13 @@ export class AddresslistVotingClientEncoding extends ClientCore
    * so that the plugin is configured
    *
    * @param {IAddresslistVotingPluginInstall} params
+   * @param {AvailableNetworks} network
    * @return {*}  {IPluginInstallItem}
    * @memberof AddresslistVotingClientEncoding
    */
   static getPluginInstallItem(
     params: IAddresslistVotingPluginInstall,
+    network: AvailableNetworks,
```

Comment:
```suggestion
    network: AvailableNetworks = "mainnet",
```

Reply:
We already do this in the client so this already has a default value there, no need of having it here too